digdag-tests/src/test/java/acceptance/RuntimeParamsIT.java

```diff
@@ -198,4 +205,55 @@ public void testParamWithSchedule()
         assertEquals("+params2_sch+t_task_name", result.get("task_name", String.class));
         assertEquals("retry01", result.get("retry_attempt_name", String.class));
     }
+
+    @Test
+    public void testbuildinParams()
```

Comment:
I guess this test should be moved to unit test.
And you should check these new parameter is really referable from workflow.
It is better to push a test workflow and run and check workflow refer to new variable correctly.
I guess you may be able refer to testParamWithSchedule().